libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -359,7 +359,8 @@ ASTPointer<UsingForDirective> ASTJsonImporter::createUsingForDirective(Json::Val
 		_node,
 		move(functions),
 		!_node.isMember("libraryName"),
-		_node["typeName"].isNull() ? nullptr  : convertJsonToASTNode<TypeName>(_node["typeName"])
+		_node["typeName"].isNull() ? nullptr  : convertJsonToASTNode<TypeName>(_node["typeName"]),
+		_node.isMember("global") && _node["global"].asBool()
```

Comment:
```suggestion
		_node.isMember("global") && memberAsBool(_node, "global")
```
?
Or only ``memberAsBool`` or is the idea to default to ``false`` instead of raising an error?

Reply:
The idea is to default to false.

Reply:
``_node.isMember("global") && memberAsBool(_node, "global")`` would still be more graceful I guess, if ``global`` is there and not a boolean, but yeah, fine either way.

Reply:
Now it *won't* default to ``false`` anymore, though, will it :-)? But yeah, it also doesn't have to, so also fine.